app/assets/stylesheets/shotlinks.scss

```diff
@@ -73,5 +113,8 @@ $info_button_blue: #5BC0DE;
 .label-success-outline {
   background-color: $backgroud_color;
   border: 1px solid #5CB85C;
-  // color: #5CB85C;
+}
+
+.text-muted {
+  color: #9AB5C2
```

Comment:
**UX-Nitpick**
Make the actual url display same color as the footer text `shotnr inc.`. This helps to make it more visible.